Simulator/simulator/plant_type.py

```diff
@@ -47,8 +47,11 @@ def in_bounds(r, c):
             name, plant = self.plant_types[np.random.randint(0, self.num_plant_types)]
             coord = coords.pop(0)
             r, c = coord[0], coord[1]
-            plants.extend([Plant(r, c, c1=plant['c1'], growth_time=plant['growth_time'],
-                                 germination_time=plant['germination_time'], color=plant['color'],
+            growth_time, c1, germination_length = generate_c1_and_growth_time(
```

Comment:
You sample some normal distributions in `generate_c1_and_growth_time()` hence `growth_time, c1, germination_length` will be different here vs simulator.plant_type.py, right? They shouldn't differ.